ocaml/driver/makedepend.ml

```diff
@@ -662,6 +662,7 @@ let run_main argv =
     let program = Filename.basename Sys.argv.(0) in
     Compenv.parse_arguments (ref argv)
       (add_dep_arg (fun f -> Src (f, None))) program;
+    List.iter Language_extension.enable Language_extension.all;
```

Comment:
Why all language extensions? Shouldn't, e.g., the parsing of `local_` be on in some cases and off in others? For example, we wouldn't want `ocamldep` to fail to parse `let local_ = "local"` in external code just because the `local_` extension is enabled.